CHANGELOG.md

```diff
@@ -1,295 +1,301 @@
+## [v0.0.35](http://github.com/ndustrialio/contxt-sdk-js/tree/v0.0.35) (2018-10-11)
+
+**Fixed**
+
+- Updated `AssetAttributes#create` and removed `organizationId` from the required fields.
```

Comment:
Might want to change this wording to something like 

Updated `AssetAttributes#create` to allow for the creation of global asset attributes.